src/genworld_gui.cpp

```diff
@@ -375,6 +389,38 @@ static DropDownList BuildTownNameDropDown()
 	return list;
 }
 
+/**
+ * Set string parameters before drawing the number of minutes per year.
+ * @param max Set the maximum value, to determine widget size.
+ */
+static void SetMinutesPerYearParams(bool max = false)
+{
+	/* When resizing widgets, the longest string is 1% of default. */
```

Comment:
I do not understand this comment. 1% of default, but you multiply the default with 100? Did you mean: the default is 1% of the longest string? I dunno, sorry :)